app/models/dispatch_load.rb

```diff
@@ -0,0 +1,31 @@
+class DispatchLoad
+  def initialize(load)
+    @load = load
+  end
+
+  def dispatch(arguments)
```

Comment:
Rename arguments to object as arguments suggest a collection of loosely defined values.